node/src/command.rs

```diff
@@ -89,6 +89,7 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
         "dolphin-dev" => Ok(Box::new(chain_specs::dolphin_development_config())),
         "dolphin-local" => Ok(Box::new(chain_specs::dolphin_local_config())),
         "dolphin-testnet" => Ok(Box::new(chain_specs::dolphin_testnet_config()?)),
+        "dolphin-testnet-ci" => Ok(Box::new(chain_specs::dolphin_testnet_ci_config()?)),
```

Comment:
Not sure you create one more branch here.
what are differences between `genesis/dolphin-testnet-ci-genesis.json` and `genesis/dolphin-testnet-genesis.json`?
I kind of forgot it.

Reply:
So I can just paste the collator mnemonics in the yml file directly for injection, they are different to the normal `-testnet` secrets, which I don't want to leak.